.rubocop_todo.yml

```diff
@@ -15,12 +15,12 @@ Layout/MultilineMethodCallBraceLayout:
     - 'spec/routing/file_routing_spec.rb'
     - 'spec/routing/media_routing_spec.rb'
 
-# Offense count: 8
+# Offense count: 9
 # Configuration parameters: AllowedMethods, AllowedPatterns, CountRepeatedAttributes.
 Metrics/AbcSize:
-  Max: 22
+  Max: 37
```

Comment:
Can we avoid this change and instead inline exceptions so we can see where they're at?

Reply:
Restored todo and inlined exception